### ISSUE TITLE
SA1513 Fired when old StyleCop did not complain

### ISSUE THREAD

**Issue by froufrou121** [DotNetAnalyzers/StyleCopAnalyzers#2041]
I like that in the old StyleCop (4.7) you could put //// comments anywhere in the code and no warning would be raised. With StyleCop.Analyzers rc3 I get a SA1513 with the following code

```csharp
            public int Do(int i)
            {
                if (i > 2)
                {
                    return 1;
                }
                //// else if (i == 2)
                //// {
                ////     return 2;
                //// }

                return 0;
            }
```

whereas I did not get any warning from StyleCop 4.7. I preferred the old way.

**Comment by sharwell** [comment]
I think it makes sense to update our code to work the same way as StyleCop Classic.

**Comment by vweijsters** [comment]
Grabbing this
